tests/unit/accounts/test_views.py

```diff
@@ -162,7 +162,7 @@ def test_returns_users_with_prefix(self, db_session, user_service):
         ]
 
         request = pretend.stub(
-            authenticated_userid=1,
+            identity=1,
```

Comment:
Tiny thing, but you might want to make this `identity=pretend.stub()` here and below to emphasize that `identity` is expected to be some kind of object model, not just an identifier 🙂